interceptors/payment_authentication.go

```diff
@@ -188,7 +187,7 @@ func PaymentAdminAuthenticationIntercept(next http.Handler) http.Handler {
 		ctx := context.WithValue(r.Context(), helpers.ContextKeyUserID, userEmail)
 
 		// Check that user has bulk refund role and sends POST request
```

Comment:
We should edit the comment to reflect that we're not checking for a POST request anymore